src/components/OptionRow.js

```diff
@@ -77,6 +77,7 @@ const propTypes = {
     isMultilineSupported: PropTypes.bool,
 
     style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    keyForList: PropTypes.string,
```

Comment:
Can you put this above style and provide a comment explaining it function as we do with other props

Reply:
@alitoshmatov I just updated